common-api/src/main/java/org/pmiops/workbench/cdr/dao/ConceptService.java

```diff
@@ -64,25 +65,36 @@ public ConceptService(EntityManager entityManager, ConceptDao conceptDao) {
         this.conceptDao = conceptDao;
     }
 
-    public static String modifyMultipleMatchKeyword(String query) {
+    public static String modifyMultipleMatchKeyword(String query, SearchType searchType) {
         // This function modifies the keyword to match all the words if multiple words are present(by adding + before each word to indicate match that matching each word is essential)
         if (query == null || query.trim().isEmpty()) {
             return null;
         }
         String[] keywords = query.split("[,+\\s+]");
         List<String> temp = new ArrayList<>();
         for (String key: keywords) {
-            if (!key.isEmpty()) {
-                if (key.contains("-") && !temp.contains(key)) {
-                    temp.add(key);
-                } else if (key.contains("*") && key.length() > 1) {
-                    temp.add(new String("+" + key));
+            String tempKey;
+            if (key.contains(".")) {
+                tempKey = "\"" + key + "\"";
```

Comment:
Found an issue when matching 100.0. By default matches with only 100 somehow. 
So matching on "100.0" (exact match).

Reply:
Could you add a comment to the code? I'm still not clear on how this relates to the SearchType

Reply:
Actually, I read the PR description and this makes some more sense (thanks for adding this) - but a comment in the code here would still be good.